scripts/importers/finish.sql

```diff
@@ -12,6 +11,12 @@ alter table newsletters add constraint uix_email_name unique (email_id, "name");
 alter table vpn_waitlist add constraint vpn_waitlist_email_id_key unique (email_id);
 alter table fxa add constraint fxa_fxa_id_key unique (fxa_id);
 
+-- Easy enough to add this now too
+create index bulk_read_index on emails (update_timestamp, email_id);
+
+-- New matching
+CREATE UNIQUE INDEX idx_email_primary_unique_email_lower ON public.emails USING btree (lower((primary_email)::text));
```

Comment:
Thanks for the reminder, I need to do some model and index cleanup. Filed https://github.com/mozilla-it/ctms-api/issues/238.